data-serving/data-service/src/util/search.ts

```diff
@@ -0,0 +1,25 @@
+import { parse, SearchParserResult } from 'search-query-parser';
+
+export interface ParsedSearch {
+    fullTextSearch?: string;
+}
+
+export default function parseSearchQuery(q: string): ParsedSearch {
```

Comment:
Could you just return a string rather than a ParsedSearch object containing a string?

Reply:
No, I'll need separate fields in that struct being returned for the extracted keywords once we start doing searches with "is:verified curator:foo@bar.com" etc.